cameo/flux_analysis/analysis.py

```diff
@@ -468,9 +580,47 @@ def data_frame(self):
         return pandas.DataFrame(self._phase_plane)
 
     def plot(self, grid=None, width=None, height=None, title=None, axis_font_size=None, palette=None,
-             points=None, points_colors=None, **kwargs):
+             points=None, points_colors=None, estimate='flux', **kwargs):
+        """plot phenotypic phase plane result
+
+        create a plot of a phenotypic phase plane analysis
+
+        Parameters
+        ----------
+        grid: plotting grid
+            the grid for plotting
+        param: int
+            the width of the plot
+        height: int
+            the height of the plot
+        title: string
+            the height of the plot
+        axis_font_size: int
+            the font sizes for the axis
+        palette: string
+            name of color palette to use, e.g. RdYlBlu
+        points: iterable of points
+            additional points to plot as x, y iterable
+        points_colors: iterable of strings
+            iterable with colors for the points
+        estimate: string
+            either flux, mass_yield (g output / g output) or c_yield (mol carbon output / mol carbon input)
+        """
+        possible_estimates = {'flux': ('objective_upper_bound',
+                                       'objective_lower_bound',
+                                       'flux product / flux source'),
+                              'mass_yield': ('mass_yield_upper_bound',
+                                             'mass_yield_lower_bound',
+                                             'gram product / gram source'),
+                              'c_yield': ('c_yield_upper_bound',
+                                          'c_yield_lower_bound',
+                                          'mol C product / mol C source')}
+        if estimate not in possible_estimates:
+            raise Exception('estimate must be one of %s' %
+                            ', '.join(possible_estimates.keys()))
+        upper, lower, description = possible_estimates[estimate]
         if title is None:
-            title = "Phenotypic Phase Plane"
+            title = "Phenotypic Phase Plane ({})".format(description)
```

Comment:
The same way you change the title, I think it will be nice to also append the units to the x and y axis labels. The assumption before was that we were working only with fluxes.